Formula/llvm.rb

```diff
@@ -225,19 +282,19 @@ def install
   end
 
   def caveats
-    s = <<-EOS.undent
+    caveat_message = <<-EOS.undent
```

Comment:
This is a [bad programming practice](http://blog.goyello.com/2013/05/17/express-names-in-code-bad-vs-clean/)